src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const man = people.filter(person => century
+    ? person.sex === 'm' && century === Math.ceil(person.died / 100)
+    : person.sex === 'm'
+  );
```

Comment:
you can use 2 filters in a row 
```suggestion
  const man = people
    .filter(person => person.sex === 'm')
    .filter(person => century ? century === Math.ceil(person.died / 100) : true);
```